### PR TITLE
podmonitor switch

### PR DESCRIPTION
**Pull request by ssyno**
## This PR:
- Adds on/off switch for the `PodMonitor`

## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: helm/irsa-operator/templates/pod-monitor.yaml [hunk 1]
```diff
@@ -14,3 +15,4 @@ spec:
   selector:
     matchLabels:
       {{- include "labels.selector" . | nindent 6 }}
+{{- end }}
```
Comment (QuentinBisson) [inline helm/irsa-operator/templates/pod-monitor.yaml:18]:
```suggestion
{{- end }}

```

**Review by QuentinBisson — APPROVED** [review]
Let's wait for @giantswarm/team-phoenix

**Comment by paurosello** [comment]
how would the podmonitor switch help if its always set to true?

**Review by paurosello — APPROVED** [review]
LGTM

**Pull request merged**